pauliopt/clifford/tableau.py

```diff
@@ -37,24 +61,24 @@ def mult_paulis(p1, p2, sign1, sign2, n_qubits):
 
 class CliffordTableau:
     """
-    Class for storing and manipulating Clifford tableau.
-    A Clifford tableau is a representation of a Clifford circuit as a
+    Class for storing and manipulating Clifford clifford.
```

Comment:
I think there are search and replace problems across the PR about clifford <-> tableau, please fix them